calour/amplicon_experiment.py

```diff
@@ -54,31 +55,35 @@ class AmpliconExperiment(Experiment):
     Attributes
     ----------
     data : numpy.ndarray or scipy.sparse.csr_matrix
-        The abundance table for OTUs, metabolites, genes, etc. Samples
+        The abundance table for OTUs or ASVs. Samples
         are in row and features in column
     sample_metadata : pandas.DataFrame
         The metadata on the samples
     feature_metadata : pandas.DataFrame
         The metadata on the features
-    exp_metadata : dict
+    metadata : dict
         metadata about the experiment (data md5, filenames, etc.)
     shape : tuple of (int, int)
         the dimension of data
     sparse : bool
         store the data as sparse matrix (scipy.sparse.csr_matrix) or dense numpy array.
     description : str
         name of the experiment
+    databases : iterable of str
+        databases for fetching and entering feature annotations
 
+    See Also
+    --------
+    Experiment
     '''
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-        self.heatmap_databases = ('dbbact',)
+    def __init__(self, *args, databases=('dbbact',), **kwargs):
+        super().__init__(*args, databases=('dbbact',), **kwargs)
 
-    def filter_taxonomy(exp: Experiment, values, negate=False, inplace=False, substring=True):
-        '''filter keeping only observations with taxonomy string matching taxonomy
+    def filter_by_taxonomy(self, values, field='taxonomy', substring=True, negate=False, inplace=False):
```

Comment:
changing parameter order breaks backwards compatibility. last time we do such a change :)